seatunnel-translation/seatunnel-translation-spark/seatunnel-translation-spark-common/src/main/java/org/apache/seatunnel/translation/spark/common/serialization/InternalRowConverter.java

```diff
@@ -74,13 +76,12 @@ private static Object convert(Object field, SeaTunnelDataType<?> dataType) {
                 SeaTunnelRowType rowType = (SeaTunnelRowType) dataType;
                 return convert(seaTunnelRow, rowType);
             case DATE:
-                return Date.valueOf((LocalDate) field);
+                return (int) ((LocalDate) field).toEpochDay();
```

Comment:
What's the meaning of using `UTC` zone? I think the data should be consistent.

Reply:
> What's the meaning of using `UTC` zone? I think the data should be consistent.
It's not important param in Tansform Processing, I choose the one casually ,and I change to Timestamp，it's GMT zone